### PR TITLE
updated recording queries to return consistent results

### PR DESCRIPTION
**Pull request by gferraro**

### CODE REVIEW

File: models/Recording.js [hunk 1]
```diff
@@ -534,44 +517,77 @@ module.exports = function(sequelize, DataTypes) {
...
+          [
+            Sequelize.fn(
+              "json_build_object",
+              "start_s",
+              Sequelize.literal(`"Tracks"."data"#>'{start_s}'`),
+              "end_s",
+              Sequelize.literal(`"Tracks"."data"#>'{end_s}'`)
+            ),
+            "data"
+          ]
```
Comment (mjs) [inline models/Recording.js]: What does it mean when you put an array in the `attributes` array like this?

File: test/test_recordings_filter.py [hunk 2]
```diff
@@ -85,8 +85,14 @@ def test_recordfing_query_tag_fields(self, helper):
             assert tracks is not None
             for track in tracks:
                 track_tags = track.get("TrackTags")
```
Comment (mjs) [inline test/test_recordings_filter.py]: This line belongs underneath the start_s and end_s asserts

File: models/Recording.js (hunk 1 above)
Comment (gferraro) [reply models/Recording.js]: ['test_data', 'test'] e.g. which would translate too "select test_data as test"

File: test/test_recordings_filter.py (hunk 2 above)
Comment (gferraro) [reply test/test_recordings_filter.py]: > Have you checked that this works with all parts of the cacophony-browse that use these APIs?

Have run through with browse, shouldn't cause any problems as it's only returning more than it was before

**Pull request merged**